### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by chriswhong**
Adds the PRs since the GL change (and one prior) to the changelog, separated by date.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "geojsonhint": "0.3.4",
     "gtfs2geojson": "^1.0.0",
     "jsonlint": "1.6.0",
+    "lodash": "^4.17.21",
```
Comment (wboykinm) [inline package.json:28]: Seems like a bit of a piggyback, but certainly a useful one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+* Add Codemirror Folding (#704)
+
+* The following updates were included in (#703)
+* Remove authentication UI and references to auth/saving.
+* Remove the browser CLI and references to it in the help text and readme.
+* Simplestyle styling is no longer supported, and default population of simplestyle properties is removed. All features in the working dataset will now have the same style
+* Upgrade browserify and other dependencies, allowing for a working build with newer versions of node.js (v14).
+* Add some vs code settings to automate running make on save, for a more streamlined development process.
+* Uses mapbox-gj.js for map rendering, including the new globe projection.
+* Adds mapbox-gl-draw with a rectangle mode and a custom edit UI/UX to be similar to the edit button in leaflet-draw
```
Comment (wboykinm) [inline CHANGELOG.md:27]: Thanks again for adding this one 🙏

**Review by wboykinm — APPROVED** [review]
LGTM 🚢

File: package.json (hunk 1 above)
Comment (chriswhong) [reply package.json:28]: nice catch, that shouldn't be in there

**Pull request closed without merging**